sympy/tensor/tensor.py

```diff
@@ -1318,6 +1341,62 @@ def __new__(cls, *args, **kw_args):
         obj = Basic.__new__(cls, base, generators, **kw_args)
         return obj
 
+    @classmethod
+    def fully_symmetric(cls, rank):
+        """
+        Returns a fully symmetric (antisymmetric if `rank`<0)
```

Comment:
the backtick will evaluate the expression as LaTeX. Maybe you should use two backticks to mark `rank` as code?

Reply:
I'm not so sure about using negative rank for antisymmetric permutations. Maybe it's better to create a separate method for antisymmetric tensors?

Reply:
I've put two backticks as you asked. Regarding the second suggestion I prefer negative values more because in the neighboring method ``direct_product(2, -2)`` means a 4-index tensor with the first 2 indexes symmetric and the last 2 antisymmetric. It is hard to do the same without the negative values...

Reply:
I see. It's not very intuitive, but it's OK.